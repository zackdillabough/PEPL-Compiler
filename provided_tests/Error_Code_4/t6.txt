POLY A(R) = R + 2 R + 1;
POLY C(Q, W, E, R, T, Y) = Q^121 + W - R^2 - 1;
POLY B(R) = R + 2 R + 1;
START
A(3);
INPUT X;
B(X);
INPUT Y;
INPUT Z;
C(X, Y, 
X, Z);
C(X);

A(1, C(2, 3, 4));
B(B(C(10), 5), B(1, 2), C(10));
9 8 6 1 3 4
