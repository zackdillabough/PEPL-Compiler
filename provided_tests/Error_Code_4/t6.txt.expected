Error Code 4: 10 12 14 14 15 15 15 15 15 